### PR TITLE
Fix IndexExprScope in TransposesOp lowering

### PR DESCRIPTION
**Pull request by tungld**
This patch fixes an issue about IndexExprScope in TransposesOp lowering and a function signature in RunONNXLib.cpp

### CODE REVIEW

File: src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp [hunk 1]
```diff
@@ -197,6 +197,7 @@ struct ONNXTransposeOpLowering : public OpConversionPattern<ONNXTransposeOp> {
     create->krnl.iterateIE(loopDef, loopDef, lbs, inUBs,
         [&](KrnlBuilder &createKrnl, ValueRange indices) {
           MultiDialectBuilder<MathBuilder, KrnlBuilder> create(createKrnl);
+          IndexExprScope loopScope(createKrnl);
```
Comment (tungld) [inline src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp:200]: Without this, it failed to create `SymbolIndexExpr` because of not the same scope.

Comment (AlexandreEichenberger) [reply src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp:200]: Sometimes I think the distinction between symbols and dims are really a nuisance, and not sure how key it is anyways... Thought on that?

**Review by AlexandreEichenberger — APPROVED** [review]
LGTM

File: src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp (hunk 1 above)
Comment (tungld) [reply src/Conversion/ONNXToKrnl/Tensor/Transpose.cpp:200]: I seems there is no difference if we just do calculation over IndexExpr, but it matters if we get an affine map from IndexExpr. If we have `a*b` in the affine map and `a` is a dim, then `b` must be a literal or symbol. If `b` is symbol, we have a semi affine map.

**Pull request merged**